qvm/sys-gui-template.sls

```diff
@@ -46,3 +46,17 @@ sys-gui-xfce:
       - xfwm4
       - xfwm4-theme-nodoka
       - xfwm4-themes
+
+sys-gui-lightdm:
+  file.managed:
+    - name: /etc/systemd/system/lightdm.service.d/qubes.conf
+    - makedirs: True
+    - contents: |
+        [Unit]
+        ConditionPathExists=/var/run/qubes-service/lightdm
+        [Install]
+        WantedBy=multi-user.target
+  cmd.run:
+    - name: systemctl daemon-reload
+  service.enabled:
+    - name: lightdm
```

Comment:
You said it doesn't work for you. Does `state.highstate` reports any errors?

Reply:
Everything executed by `state.highstate` passes. The problem is the override defined just above doesn't seem to be recognized. The symlink to `lightdm.service` file is **not** created in `/etc/systemd/system/multi-user.target.wants`
Manually executing `systemctl enable lightdm` works as expected.

Reply:
`Service lightdm is alrady enabled and is in the desired state` :/